src/services/JobRoleService.ts

```diff
@@ -0,0 +1,12 @@
+import axios, { AxiosResponse } from "axios";
+import { JobRoleResponse } from "../models/JobRoleResponse";
+
+export const getJobRoles = async (): Promise<JobRoleResponse[]> => {
+    try {
+        const response: AxiosResponse = await axios.get("http://localhost:8080/api/job-roles"); // will need amended
```

Comment:
this will work locally but not on AWS, please pass the base API URL in as an environment variable